### DISCUSSION TITLE
Re-importing ebooks (PDF & EPUB formats) back to Calibre after deletion

### DISCUSSION THREAD

**Discussion by Rat-a-Tat** [noDRM/DeDRM_tools#185] [Q&A]
I recently imported some ebooks in pdf and pub formats to Calibre after running them through Adobe Digital Editions.

I added them to another ebook library and deleted them from the Calibre directory.

I'm wanting to add them back to Calibre so I can use its "cloud" option on all my devices, but I'm running into trouble since Calibre is unable to "read" the files and populate the book information correctly as it imports them.

I thought I would ask this question here since maybe I am missing a step in the import process. I've searched the forum but didn't find any leads...

Thanks!

(I hope I am asking this question in the right place on this site)

**Comment by ElleKayEm** [comment]
I'd ask over in the calibre forum at MobileRead.com if this is unrelated to DRM removal problems.
